tests/test_smoke.py

```
@@ -0,0 +1,142 @@
+
+    def click(self, *args, **kwargs):
+        pass
+
+    def hotkey(self, *args, **kwargs):
+        pass
+
+
+sys.modules.setdefault("pyautogui", DummyPyAutoGUI())
+
+# ---- Monkeypatch LMMAgent to avoid external LLM calls ----
+import gui_agents.s3.core.mllm as mllm  # noqa: E402
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)